backend/bin/deal-observer-backend.js

```diff
@@ -8,7 +8,7 @@ import '../lib/instrument.js'
 import { createInflux } from '../lib/telemetry.js'
 import { getChainHead, rpcRequest } from '../lib/rpc-service/service.js'
 import { fetchDealWithHighestActivatedEpoch, countStoredActiveDeals, observeBuiltinActorEvents } from '../lib/deal-observer.js'
-import { lookUpPayloadCids } from '../lib/look-up-payload-cids.js'
+import { countStoredActiveDealsWithUnresolvedPayloadCid, lookUpPayloadCids } from '../lib/look-up-payload-cids.js'
```

Comment:
If you look at this line, this is inconsistent naming: In the first function, we call it "resolve payload cid", in the 2nd it's "look up payload cid". Inconsistencies like this make the code harder to reason about, therefore it should be either or.

Suggestion: Go another way, and name like this instead:

```suggestion
import { countStoredActiveDealsWithoutPayloadCid, lookUpPayloadCids } from '../lib/look-up-payload-cids.js'
```

or

```suggestion
import { countStoredActiveDealsMissingPayloadCid, lookUpPayloadCids } from '../lib/look-up-payload-cids.js'
```

Reply:
Hm I did this intentional. I thought one thing is the actionable word `look up` and the other addresses the actual state of the payload `unresolved`. I don't quite see the inconsistency here.

Reply:
If we use "unresolved", we should use "resolve()" instead of "lookup()". Resolving is the fix for something being unresolved.

Reply:
Given the changes that follow in the [missing indexer PR](https://github.com/CheckerNetwork/spark-deal-observer/issues/30) I think using resolve makes more sense here. But I would do this in a separate PR as it is not scope of #84

Reply:
>Given the changes that follow in the https://github.com/CheckerNetwork/spark-deal-observer/issues/30

Not sure I follow, you linked to an issue not a PR?